Snippets/SqlPersistence/SqlPersistence_6/OutboxSettings.cs

```diff
@@ -44,6 +45,19 @@ void OutboxSettingsEx4(EndpointConfiguration endpointConfiguration)
         var outboxSettings = endpointConfiguration.EnableOutbox();
 
         outboxSettings.UseTransactionScope();
+        outboxSettings.UseTransactionScope(IsolationLevel.RepeatableRead);
```

Comment:
Does this mean both lines will appear in the snippet? Should there be an `// OR` in there or something?
```suggestion
        // OR
        outboxSettings.UseTransactionScope(IsolationLevel.RepeatableRead);
```